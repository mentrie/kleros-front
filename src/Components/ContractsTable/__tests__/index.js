import React from 'react'
import ContractTable from '../index'

describe('ContractTable Exist', () => {
  const component = shallow(<ContractTable />)
  test('should render component properly', () => {
    expect(component).toMatchSnapshot()
  })

  test('should render button with text', () => {
    expect(component.find('button').text()).toBe('New Contract')
  })
})
