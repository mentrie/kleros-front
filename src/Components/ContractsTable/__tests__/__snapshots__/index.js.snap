// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContractTable Exist should render component properly 1`] = `
<div
  className="ContractsTable-container"
>
  <SearchBar />
  <div
    className="content"
  >
    <h1>
      Open Contracts
      <span
        className="pull-right"
      >
        <Link
          key="new-contract"
          replace={false}
          to="contracts/new"
        >
          <button
            className="submit"
          >
            New Contract
          </button>
        </Link>
      </span>
    </h1>
    <Grid>
      <withRouter(Connect(GridContent)) />
    </Grid>
  </div>
</div>
`;
